test/adaptive_load/fake_plugins/fake_input_variable_setter/fake_input_variable_setter.cc

```diff
@@ -0,0 +1,70 @@
+#include "test/adaptive_load/fake_plugins/fake_input_variable_setter/fake_input_variable_setter.h"
+
+namespace Nighthawk {
+
+namespace {
+
+absl::Status StatusFromProtoRpcStatus(const google::rpc::Status& status_proto) {
+  return absl::Status(static_cast<absl::StatusCode>(status_proto.code()), status_proto.message());
+}
+
+} // namespace
+
+FakeInputVariableSetter::FakeInputVariableSetter(
+    const nighthawk::adaptive_load::FakeInputVariableSetterConfig& config)
+    : adjustment_factor_{config.adjustment_factor() > 0 ? config.adjustment_factor() : 1} {}
+
+absl::Status FakeInputVariableSetter::SetInputVariable(
+    nighthawk::client::CommandLineOptions& command_line_options, double input_value) {
+  command_line_options.mutable_connections()->set_value(static_cast<unsigned int>(input_value * adjustment_factor_));
```

Comment:
I wonder -- what happens when input value is negative? Do we need special handling? Is this covered in a test? If it is not possible to get negative input, maybe assert here?

Reply:
Sorry in retrospect this commebr seems overly strict as I just realize this is test code; but maybe this is worth broader consideration?

Reply:
Just made it uint32 since we don't need negative values.